### ISSUE TITLE
Spacebar Swipe Option for Cursor Positioning

### ISSUE THREAD

**Issue by jeebsdotorg** [klausw/hackerskeyboard#753]
Similar to the title of #243, it would be great to move the input cursor around left-to-right by simply swiping the spacebar.  Currently, this action looks to be solely responsible for changing the soft-keyboard input layout instead.  This is a killer feature of Gboard, in my opinion.  It is much rarer for me to change languages.  I also use mostly portrait mode now-a-days, so horizontal orientation arrow keys are not so helpful.

**Comment by KevinBlue18** [comment]
You can put arrow keys in portrait mode too:

**Comment by jeebsdotorg** [comment]
That is true.  The 4 row Gingerbread layout is preferable, due to the ever-so-slightly bigger keys.  Otherwise, I end up with way too many typos.  I just think a setting to mimic Gboard, in this case ,would be a great enhancement, is all.
